### PR TITLE
{geo}[foss/2015a] WRF 3.6.1

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 13 out of 13 (12 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3d012043d768b69c89e9 for a full test report.

File: easybuild/easyconfigs/c/CMake/CMake-3.0.0-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.0.0'
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/CMake/CMake-3.0.0-foss-2015a.eb]: there is no reason not to upgrade this to 3.2?

File: easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name = 'WRF'
+version = '3.6.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
```
Comment (wpoely86) [inline easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb]: Isn't `opt` set to `False` by default? And does this automatically fall back to `-O2`?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/c/CMake/CMake-3.0.0-foss-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/CMake/CMake-3.0.0-foss-2015a.eb]: I prefer keeping versions in sync across toolchains, this is a dep for netCDF 4.3.2, and we already have easyconfigs for this one

File: easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb]: No, `opt` (which corresponds to `-O3`) is the default. Disabling `opt` implicitly makes `defaultopt` the active one, which corresponds to `-O2`.

See https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/tools/toolchain/compiler.py#L101

Yes, we need better documentation for this, or a dedicated cmdline option to query this.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4550ca5732a94e0731ce for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**